### PR TITLE
Carlos taylor - Assignment 1 submission

### PR DESCRIPTION
**Pull request by CarlosTFS**
This is my 1st attempt at assignment 1

### CODE REVIEW

File: Assignment 1/CarlosTaylor.txt
Comment (Helcostr) [inline Assignment 1/CarlosTaylor.txt:1]: This is a good job

File: .gitignore
Comment (Helcostr) [inline .gitignore:1]: You deleted this file by accident.

File: README.md
Comment (Helcostr) [inline README.md:1]: You deleted this file by accident.

**Review by Helcostr — CHANGES_REQUESTED** [review]
Good job, but you deleted two files by accident. Please check and try again.

**Comment by CarlosTFS** [comment]
I have added the missing files

**Review by Helcostr — APPROVED** [review]
Good job!

**Pull request merged**